src/Stratis.Bitcoin.Tests/Consensus/CheckpointsTest.cs

```diff
@@ -131,7 +132,7 @@ public void GetCheckPoint_CheckpointExists_PoWChain_ReturnsCheckpoint()
         [Fact]
         public void GetCheckPoint_CheckpointExists_PoSChain_ReturnsCheckpoint()
         {
-            var checkpoints = new Checkpoints(Network.StratisMain, new ConsensusSettings() { UseCheckpoints = true });
+            var checkpoints = new Checkpoints(Network.StratisMain, new ConsensusSettings(new NodeSettings()) { UseCheckpoints = true });
```

Comment:
Instead of having to specify `new NodeSettings()` inside every *Settings instantiation, would it make sense to have a parameterless ctor call `new NodeSettings()` or `NodeSettings.Default()`?

Reply:
Fixed